app-modules/ai/tests/Tenant/Feature/Filament/Resources/AiAssistantResource/EditAiAssistantTest.php

```diff
@@ -149,7 +149,7 @@
     assertDatabaseHas(
         AiAssistant::class,
         $request->except([
-            'avatar',
+            'avatar', 'model',
```

Comment:
Does this fail without this added right now? If so, then I believe there is a problem. Because `model` should still be being populated and saved even if the User is not an admin.

It should probably default to what the default institutional model is.